db/seeds.rb

```diff
@@ -1,20 +1,12 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
-
 db = Specialty.create(name: "Database",
         url_name: "database",
         image: File.new("#{Rails.root}/app/assets/images/database/horse_animal_totem.png"))
 fe = Specialty.create(name: "Front End",
         url_name: "front_end",
-        image: File.new("#{Rails.root}/app/assets/images/back_end/hawk_animal_totem.png"))
+        image: File.new("#{Rails.root}/app/assets/images/front_end/hawk_animal_totem.png"))
```

Comment:
Line is too long. [91/80]<br>Align the elements of a hash literal if they span more than one line.